specification/glossary.md

```diff
@@ -104,6 +104,10 @@ A company or organization that provides outsourced management and support of a r
 
 A term that describes a service that is available and provided immediately or as needed, without requiring a pre-scheduled appointment or prior arrangement. In cloud computing, virtual machines can be created and terminated as needed, i.e. on demand.
 
+<a name="glossary:negotiated-discount"><b>Negotiated Discount</b></a>
```

Comment:
We should differentiate in Commitment discount related columns that it is not meant for the negotiated discounts (this is what we do for consumed vs pricing units - we explicitly state that one is for pricing and cost and the other is for consumption). Similar differentiation in commitment discount columns using this glossary entry will be useful.

Reply:
Applied ~"... confused with negotiated discounts ..." across all `CommitmentDiscount*` columns